### PR TITLE
Install Python in Conda Environment creation with no packages

### LINKED ISSUE #4799: Add more input validation to add conda environment
The "no packages" in particular will probably cause confusion. This will lead to an environment without python, the user will wonder why it doesn't appear in the environment window.

It will be difficult without waiting for the conda preview to determine if the environment will have python or not, but one thing is for sure is that if packages text box is empty, it will not have python.

### PR DESCRIPTION
**Pull request by RaymonGulati1**
Fix #4799

### CODE REVIEW

File: Python/Product/Common/Strings.resx [hunk 1]
```diff
@@ -2441,7 +2441,7 @@ Expand selection to the full expression?</value>
     <value>On_e or more Anaconda package names</value>
   </data>
   <data name="AddCondaPackagesWatermark" xml:space="preserve">
-    <value>ex. numpy pandas</value>
```
Comment (huguesv) [inline Python/Product/Common/Strings.resx]: lowercase was correct, please undo

File: Python/Product/Common/Strings.resx [hunk 2]
```diff
@@ -2450,7 +2450,7 @@ Expand selection to the full expression?</value>
     <value>Add packages</value>
   </data>
   <data name="AddCondaPackagesHelpText" xml:space="preserve">
-    <value>List of Conda packages to install when creating the Conda environment. Entries should be separated by spaces. Example: numpy pandas</value>
+    <value>List of Conda packages to install when creating the Conda environment. Entries should be separated by spaces. Example: Numpy Pandas</value>
```
Comment (huguesv) [inline Python/Product/Common/Strings.resx]: same - lowercase was correct

Comment (RaymonGulati1) [reply Python/Product/Common/Strings.resx]: Fixed

File: Python/Product/Common/Strings.resx (hunk 1 above)
Comment (RaymonGulati1) [reply Python/Product/Common/Strings.resx]: Fixed

**Pull request merged**